server/package.json

```diff
@@ -19,65 +19,64 @@
   },
   "dependencies": {
-    "@nestjs/common": "^7.3.2",
-    "@nestjs/core": "^7.3.2",
+    "@nestjs/common": "^7.4.4",
+    "@nestjs/core": "^7.4.4",
     "@nestjs/jwt": "^7.1.0",
     "@nestjs/passport": "^7.1.0",
-    "@nestjs/platform-fastify": "^7.3.2",
-    "@nestjs/platform-socket.io": "^7.3.2",
-    "@nestjs/swagger": "^4.5.12",
-    "@nestjs/typeorm": "^7.1.0",
-    "@nestjs/websockets": "^7.3.2",
-    "@sentry/node": "^5.19.1",
-    "argon2": "^0.26.2",
-    "aws-sdk": "^2.713.0",
+    "@nestjs/platform-fastify": "^7.4.4",
+    "@nestjs/platform-socket.io": "^7.4.4",
+    "@nestjs/swagger": "^4.6.1",
+    "@nestjs/typeorm": "^7.1.4",
+    "@nestjs/websockets": "^7.4.4",
+    "@sentry/node": "^5.24.2",
+    "argon2": "^0.27.0",
+    "aws-sdk": "^2.766.0",
     "class-transformer": "^0.3.1",
-    "class-validator": "^0.11.1",
+    "class-validator": "0.11.1",
     "concat-stream": "^2.0.0",
     "dotenv": "^8.2.0",
-    "ejs": "^3.1.3",
-    "fastify-compress": "^2.0.1",
-    "fastify-cookie": "^3.6.1",
-    "fastify-helmet": "^3.0.2",
-    "fastify-metrics": "^6.0.1",
-    "fastify-multipart": "^1.0.5",
-    "fastify-swagger": "^2.6.0",
+    "ejs": "^3.1.5",
+    "fastify-cookie": "^4.1.0",
+    "fastify-helmet": "^5.0.3",
+    "fastify-metrics": "^6.0.2",
+    "fastify-multipart": "^3.2.0",
+    "fastify-swagger": "^3.4.0",
     "glob": "^7.1.6",
     "ioredis": "^4.17.3",
     "jsonwebtoken": "^8.5.1",
     "keymachine": "2.1.0",
-    "mongodb": "^3.5.9",
+    "mongodb": "^3.6.2",
     "nan": "^2.14.1",
-    "nest-raven": "^5.0.0",
+    "nest-raven": "^7.0.0",
     "nest-router": "^1.0.9",
-    "nestjs-rate-limiter": "^2.5.2",
-    "nodemailer": "^6.4.10",
+    "nestjs-rate-limiter": "2.5.3",
+    "nodemailer": "^6.4.13",
     "passport": "^0.4.1",
     "passport-jwt": "^4.0.0",
     "reflect-metadata": "^0.1.13",
-    "rxjs": "^6.6.0",
-    "slugify": "^1.4.4",
-    "typeorm": "^0.2.25",
+    "rxjs": "^6.6.3",
+    "slugify": "^1.4.5",
+    "typeorm": "^0.2.28",
     "xml-js": "^1.6.11"
   },
   "devDependencies": {
-    "@nestjs/cli": "^7.4.1",
-    "@nestjs/schematics": "^7.0.1",
-    "@nestjs/testing": "^7.3.2",
+    "@nestjs/cli": "^7.5.1",
+    "@nestjs/schematics": "^7.1.2",
+    "@nestjs/testing": "^7.4.4",
     "@types/dotenv": "^8.2.0",
-    "@types/jest": "^26.0.4",
-    "@types/node": "^14.0.22",
-    "@types/socket.io": "^2.1.8",
+    "@types/jest": "^26.0.14",
+    "@types/node": "^14.11.2",
+    "@types/socket.io": "^2.1.11",
     "@types/supertest": "^2.0.10",
-    "jest": "^26.1.0",
-    "prettier": "^2.0.5",
-    "supertest": "^4.0.2",
-    "ts-jest": "^26.1.1",
-    "ts-loader": "^8.0.0",
-    "ts-node": "^8.10.2",
+    "jest": "^26.4.2",
+    "prettier": "^2.1.2",
+    "supertest": "^5.0.0",
+    "ts-jest": "^26.4.1",
+    "ts-loader": "^8.0.4",
+    "ts-node": "^9.0.0",
     "tsconfig-paths": "^3.9.0",
     "tslint": "^6.1.2",
-    "typescript": "^3.9.6"
+    "typescript": "^4.0.3"
```

Comment:
it's not good to implement a business with lots of packages of version change,  one mission for one PR that is great for review and mange code.